Lesson4/src/app/components/WelcomeModal.js

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+
+export default class WelcomeMode extends React.Component {
+    constructor(props) {
+        super(props);
+        this.state = {
+            isOpen = true
```

Comment:
```suggestion
            isOpen = false
```
чтобы в `componentDidMount()` переключить на `true`